lib/js/components/ProgressDonutChart/ProgressDonutChart.vue

```diff
@@ -0,0 +1,287 @@
+				cx="20"
+				cy="20"
+				r="18"
+			/>
+			<template v-else>
+				<circle
+					v-for="(range, index) in calculatedRanges"
+					:key="`circle_${index}`"
+					class="progressDonutChart__circle progressDonutChart__track"
+					cx="20"
+					cy="20"
+					r="18"
+					:style="`--length: ${range.length}; transform: rotate(calc(90deg + ${range.rotate}deg));`"
+					:class="[`-${range.color}`]"
+				/>
+			</template>
+		</svg>
+		<div class="progressDonutChart__label">
+			<div
+				v-if="state === PROGRESS_DONUT_CHART_STATES.LOADING"
+				class="progressDonutChart__loaderText"
+			/>
+			<div
+				v-else-if="state === PROGRESS_DONUT_CHART_STATES.DONE"
+				class="progressDonutChart__icon"
+			>
+				<ds-icon :icon="ICONS.FA_CHECK_SOLID" :size="ICON_SIZES.X_SMALL" />
+			</div>
+			<div
+				v-else-if="label"
+				class="progressDonutChart__labelText"
+				:class="{ '-success': state === PROGRESS_DONUT_CHART_STATES.OVERAGE }"
+			>
+				{{ labelText }}
+			</div>
+		</div>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../styles/settings/animations';
+@import '../../../styles/settings/colors/tokens';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/typography/tokens';
+
+$progress-donut-chart-width: 40px;
+$progress-donut-chart-height: 40px;
+$progress-donut-chart-circle-stroke-width: 4;
+$pi: 3.1416;
+$progress-donut-chart-circle-radius: 18; // (width / 2) - (stroke-width / 2)
+$circle-circumference: 2 * $pi * $progress-donut-chart-circle-radius;
+$progress-donut-chart-loading-dot-width: 2px;
+$progress-donut-chart-loading-dot-height: 2px;
+
+$progress-donut-chart-range-colors: (
+	'primaryMedium': $color-primary-data-medium,
+	'primary': $color-primary-data,
+	'primaryWeak': $color-primary-data-weak,
+	'primaryGhost': $color-primary-data-ghost,
+	'neutralMedium': $color-neutral-data-medium,
+	'neutral': $color-neutral-data,
+	'neutralWeak': $color-neutral-data-weak,
+	'neutralGhost': $color-neutral-data-ghost,
+	'infoMedium': $color-info-data-medium,
+	'info': $color-info-data,
+	'infoWeak': $color-info-data-weak,
+	'infoGhost': $color-info-data-ghost,
+	'successMedium': $color-success-data-medium,
+	'success': $color-success-data,
+	'successWeak': $color-success-data-weak,
+	'successGhost': $color-success-data-ghost,
+	'warningMedium': $color-warning-data-medium,
+	'warning': $color-warning-data,
+	'warningWeak': $color-warning-data-weak,
+	'warningGhost': $color-warning-data-ghost,
+	'failMedium': $color-fail-data-medium,
+	'fail': $color-fail-data,
+	'failWeak': $color-fail-data-weak,
+	'failGhost': $color-fail-data-ghost,
+);
+
+@keyframes dot-flashing {
+	0% {
+		background-color: $color-neutral-text;
+	}
+
+	50%,
+	100% {
+		background-color: transparent;
+	}
+}
+
+.progressDonutChart {
+	position: relative;
+	width: $progress-donut-chart-width;
+	height: $progress-donut-chart-height;
+	padding: 0;
+	margin: 0;
+	overflow: hidden;
+	box-sizing: border-box;
```

Comment:
```suggestion
	overflow: hidden;
```

Why do we need this attributes?